scheduler/tests/conftest.py

```diff
@@ -1,49 +1,54 @@
+import os
 import time
-import uuid
 from unittest import mock
-
-import sqlalchemy
-from sqlalchemy import orm
+from unittest.mock import patch
 
 import pytest
 import sqlalchemy_utils
 import tenant_dependency
+from alembic import command
+from alembic.config import Config
 from fastapi import testclient
-from scheduler import app
 from scheduler.db import models
+from scheduler.db.service import get_test_db_url
+from sqlalchemy import create_engine, orm
+from sqlalchemy.exc import SQLAlchemyError
+
+from scheduler import app
+from scheduler import config as scheduler_config
+
+alembic_cfg = Config("alembic.ini")
+
+
+@pytest.fixture(scope="session")
+def use_temp_env_var():
+    with patch.dict(os.environ, {"USE_TEST_DB": "1"}):
+        yield
 
 
 @pytest.fixture
 def testing_engine():
-    url = "sqlite:///test.db"
-    engine = sqlalchemy.create_engine(url)
-    # Replace postgres columns such UUID and JSONB for sqlite compatible.
-    models.Base.metadata.tables["units"].append_column(
-        sqlalchemy.Column("runner_id", sqlalchemy.String(36)),
-        replace_existing=True,
-    )
-    models.Base.metadata.tables["heartbeat"].append_column(
-        sqlalchemy.Column(
-            "id",
-            sqlalchemy.String(36),
-            primary_key=True,
-            default=lambda: str(uuid.uuid4()),
-        ),
-        replace_existing=True,
-    )
-    yield engine
+    db_url = get_test_db_url(scheduler_config.DB_URL)
+    test_db_engine = create_engine(db_url)
+
+    yield test_db_engine
     for _ in range(20):
         try:
-            sqlalchemy_utils.drop_database(url)
+            sqlalchemy_utils.drop_database(db_url)
             break
         except PermissionError:
             time.sleep(0.001)
 
 
 @pytest.fixture
-def testing_session(testing_engine):
+def testing_session(use_temp_env_var, testing_engine):
     sqlalchemy_utils.create_database(testing_engine.url)
-    models.Base.metadata.create_all(bind=testing_engine)
+
+    try:
+        command.upgrade(alembic_cfg, "head")
+    except SQLAlchemyError as e:
+        raise SQLAlchemyError(f"Got an Exception during migrations - {e}")
```

Comment:
Why should we raise the same exception with slightly adjusted message and lost traceback?

Reply:
By doing this I give an understanding, that this Exception is raised specifically during migrations in explicit way.
It helped me a lot during development

Reply:
```suggestion
    except SQLAlchemyError as e:
        raise SQLAlchemyError("Exception during migrations") from e
```

Reply:
Agreed - applied your suggestion